pkg/skaffold/docker/parse.go

```diff
@@ -442,3 +383,10 @@ func hasMultiStageFlag(flags []string) bool {
 	}
 	return false
 }
+
+func changeDir(cur, to string) string {
```

Comment:
It's hard to understand what this function is doing. It doesn't change the directory.

Reply:
I renamed it to `resolveDir` and added some godoc.